src/pages/ProtocolPage.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import ProtocolOptions from "../components/ProtocolOptions";
+
+const styles = `
+    .row {
+        width: 100%;
+        height: 0vh;
+    }
+
+    .protocol-wrapper {
+        background-color: rgba(234, 234, 234, 1);
+        height: 100vh;
+    }
+
+    .protocol-number {
+        background-color: rgba(245, 148, 137, 1);
+        max-width: 85px;
+    }
+
+    .input-name {
+        border: 0px;
+        width: 150px;
+    }
+
+    .font-barlow {
+        font-family: 'Barlow', sans-serif;
+    }
+`;
+
+function ProtocolPage(props) {
+    return ( 
+        <div>
+            <div className="protocol-wrapper d-flex px-4 py-4">
+                <div className="row align-items-center">
```

Comment:
```suggestion
                <div className="row align-items-center m-0">
```

- Por padrão, as rows do Bootstrap tem margens que eventualmente fazem ela "cair" da div pai. Nesse caso, o componente ficou torto pra esquerda. Zerar essas margens deve resolver;